### PR TITLE
Bug 1180651 - Create a job to run gaia-ui-tests against a Spark build…

### PR DESCRIPTION
**Pull request by JohanLorenzo**
… (on Flame)

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9230ac30f89018e99bd2dd44a076b33243c58d67

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7aa8a52d42a14f2aaf0ec1a6aafa98eb611df543

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9b8981a5493612c280d2a2062e69d1edbdbb21b2

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=eef0893904e17253125d01e5312583fbffff1a31

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_spark_apps_present.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+class TestSparkAppsArePresent(GaiaTestCase):
+
+    def test_spark_apps_are_present(self):
+        homescreen = Homescreen(self.marionette)
+        self.apps.switch_to_displayed_app()
+        for expected_app in ('Customizer', 'Hackerplace', 'Studio', 'Sharing', 'Webmaker', 'Bugzilla Lite', 'Facebook',
+                             'Twitter', 'RunWhatsApp (Preview)', 'BuddyUp', 'Notes', 'Calculator', 'SWOOOP',
+                             'Firesea IRC'):
+            homescreen.wait_for_app_icon_present(expected_app)
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_spark_apps_present.py]: Return?

Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_spark_apps_present.py]: Sorry I'm not sure to understand the question. We don't need neither an `assert` nor a `return` here. If one of these apps is not here, an exception will be thrown by the Wait() method making the test fail.

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_spark_apps_present.py]: Sorry, I meant you need to add an extra line with a return here, as I think the convention is that you need to end with an empty line, right? (sorry, this was really not worth a discussion over)

Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_spark_apps_present.py]: You're right, it's the convention in git. The file does end with an end of line. If it didn't, you'd see a red sign at the end of the line in Github.

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_spark_apps_present.py]: Oops!

**Pull request merged**